src/foam/nanos/crunch/Capability.js

```diff
@@ -326,6 +318,12 @@ foam.CLASS({
       Disables crunchService.isRenewable from propogating anything else in the heirarchy.
       Disables user expiry notifications.
       Disables capability.maybeReOpen()`
+    },
+    {
+      class: 'foam.core.Enum',
+      of: 'foam.nanos.auth.LifecycleState',
+      name: 'lifecycleState',
+      value: 'ACTIVE'
```

Comment:
Can we add documentation here on what we are expecting these states to be. A place to reference our expectations.

perhaps...
documentation: `
  Enum value interpretation:
   Pending  - Capability does not return on find or select
   Active     -  Capability does return on find or select
   Rejected - Capability does not return on find or select
   Deleted   - Capability does not return on find or select
   `

Reply:
Pending/Active work to replace Enabled. 
  Pending - ! enabled.
  Active - enabled.
Rejected - unused. 
Deleted - deleted - but this is a case where we really don't need to 'mark' deleted, but no way around that with Lifecycle.

Reply:
okidokes